### DISCUSSION TITLE
osu!std argon skin slider ends - When there's a very short, repeating slider, the repeating symbol cannot be read.

### DISCUSSION THREAD

**Discussion by akai-hana** [ppy/osu#22048] [Ideas] [answered]
![photo](https://user-images.githubusercontent.com/55887501/211049358-4d619ffb-933f-4af9-b600-204778790750.png)
Map: https://osu.ppy.sh/beatmapsets/103691#osu/1288056 (the very beginning of the map)

As you can see by the photo, the difference between short, repeating sliders and regular hitcircles is not very noticeable, which makes reading a somewhat common mapping element harder to read.

![photo-2](https://user-images.githubusercontent.com/55887501/211050026-26c88b1a-20f2-4967-b04f-61a5d958e999.png)
Same map. Skin: re;owoTuna v1.0 [Selyu]

However, checking this second photo, noticing that the slider repeats is much more noticeable, both because the repeat symbol at the end of the slider is visible, and because there's also a slider-end circle surrounding the, err, end of the slider, forgive the redundancy.

My suggestion would simply be to implement one (or both) of these elements into a future argon skin update.

Ty for reading.

**Comment by mk56-spn** [comment]
The current design if the slider end is an MVP. https://www.figma.com/file/6m10GiGEncVFWmgOoSyakH/osu!-Figma-Library?node-id=0%3A1 as you can see if you go here and click on the " gameplay " section. Iirc it was implemented in this simpler way at the time because of technical constraints regarding the shape of the final design. But it will come with time. 

I agree that it should probably extend past the sliders radius line like this skin though.

**Comment by peppy** [comment, marked as answer]
Thanks for your feedback, we're aware of this.
